### ISSUE TITLE
Release distributions don't include license file

### ISSUE THREAD

**Issue by michellehirsch** [mathworks/climatedatastore#27]
The MLTBX, which is the primary distribution vehicle for this toolbox, doesn't include the license file. The license is only available for users who download the entire repo, not installation via File Exchange or Add-On Explorer

**Comment by rpurser47** [comment]
Agreed.  The `LICENSE.txt` should get added to the MLTBX before release.
